docs/elements/hx-search/index.html

```diff
@@ -18,76 +20,43 @@
     <dl class="metadata hxList">
       <div>
         <dt>Permitted Parents</dt>
-        <dd>any</dd>
+        <dd><a href="elements/hx-search-control">&lt;hx-search-control&gt;</a></dd>
+      </div>
+      <div>
+        <dt>Permitted Siblings</dt>
```

Comment:
@CITguy, I'm proposing a way to document the `<hx-search>` facade.  Also, we'll have to update the other basic form controls with a facade.

<img width="793" alt="Screen Shot 2019-10-16 at 12 11 46 PM" src="https://user-images.githubusercontent.com/10120600/66944875-a8e2ee00-f013-11e9-8dba-91e859687691.png">

Reply:
Please use current format (see previous comment about formatting).

Reply:
I agree the format above it not idea, though please see my response to your comments above   in https://github.com/HelixDesignSystem/helix-ui/pull/620#discussion_r337171261.